xpk.py

```diff
@@ -1114,7 +1210,130 @@ def run_gke_cluster_create_command(args) -> int:
   return 0
 
 
-def create_cluster_configmap(args, system):
+def set_up_cluster_network(args) -> int:
```

Comment:
These functions are specific for H100 GPUs right? Or for all GPUs? Can you add some context in the function name to help with clarity?


Great docstring function formats btw!